.github/workflows/clang-tidy-modern.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+run-clang-tidy-10.py -config='' -header-filter='^((?!.*kiss_fft\.h.*).)*$,.*' -fix
```

Comment:
Please add a reason for excluding kiss_fft files

Reply:
Working my way up PCL's [dependency graph](https://user-images.githubusercontent.com/1709142/87282880-fd5cc080-c4f4-11ea-878e-9c7de9c71eb8.png), this PR hasn't gotten to the `kiss_fft` level.

Reply:
In that case, I'd recommend adding a comment like:
```bash
# ignoring kiss_fft, leaving it for future PR. Tracked in Issue #<issue here>
```